src/main/resources/static/js/autocomplete.js

```diff
@@ -1,61 +1,49 @@
-let autocompleteInstances = {}; // 各インスタンスの保持
-let sessionToken;
-
-// Autocompleteの初期設定
-function InitAutocomplete() {
+class SessionTokenManager {
     sessionToken = new google.maps.places.AutocompleteSessionToken();
+    regenerateSessionToken() {
+        this.sessionToken = new google.maps.places.AutocompleteSessionToken();
+    }
+}
 
-    // startPlaceとendPlaceに対するAutocompleteの設定
-    SetupAutocomplete('startPlace');
-    SetupAutocomplete('endPlace');
+let sessionManager;
 
-    const placeInputs = document.querySelectorAll("[id^='place']");
-    placeInputs.forEach((input) => {
-        SetupAutocomplete(input.id);
-    });
+function InitAutocomplete() {
+    sessionManager = new SessionTokenManager();
+
+    const initialInstance = [
+        'startPlace',
+        'endPlace',
+        'place1'
+    ];
+
+    initialInstance.forEach((instanceId) => {
+        const inputElement = document.getElementById(instanceId);
+        if (inputElement) {
+            setupAutocomplete(inputElement);
+        }
```

Comment:
```suggestion
        if (!inputElement) return;
        
        setupAutocomplete(inputElement);
```
ネストはあればあるほど見づらく、なければないほど見やすくなるため、
いかにネストしないかを考えると良き

Reply:
ここで合わせて修正されているためOK
https://github.com/saitokazumasa/tabisketch/pull/141/commits/470a51788ddab85722a5c60e16d7ee0b8151e52a